internal/pkg/manifest/workload.go

```diff
@@ -49,15 +49,22 @@ var (
 	// WorkloadTypes holds all workload manifest types.
 	WorkloadTypes = append(ServiceTypes, JobTypes...)
 
-	// All placement options.
-	subnetPlacements = []string{PublicSubnetPlacement, PrivateSubnetPlacement}
+	// Minimum CPU and mem values required for Windows-based tasks.
+	WindowsTaskCPU    = 1024
+	WindowsTaskMemory = 2048
```

Comment:
```suggestion
	MinWindowsTaskCPU    = 1024
	MinWindowsTaskMemory = 2048
```
(we can also move these to `const` instead of `var`)